task_list

```diff
@@ -0,0 +1,2 @@
+Napisz test automatyczny, który zaloguje się na stronie https://www.saucedemo.com i doda 3 randomowe rzeczy do koszyka.
+Test powinien sprawdzić czy kwota podliczona przy checkoucie jest prawidłowa
```

Comment:
Please translate it to english and change the file extension to .txt

Reply:
Fixed